### PR TITLE
AlignOps EndianOps cpp files

### PR DESCRIPTION
**Pull request by nolan-veed**
# Why

For https://github.com/roc-streaming/roc-toolkit/issues/601

# What

* Moved non-template methods to cpp files.
* Removed inline as a result.

# Testing

There are tests.

### CODE REVIEW

File: src/internal_modules/roc_core/align_ops.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
  */
 
 //! @file roc_core/align_ops.h
-//! @brief Alignment operations.
+//! @brief TODO.
```
Comment (gavv) [inline src/internal_modules/roc_core/align_ops.h:10]: Accidental change, I guess

**Review by gavv — COMMENTED** [review]
Thanks for PR! One small comment

File: src/internal_modules/roc_core/align_ops.h (hunk 1 above)
Comment (gavv) [reply src/internal_modules/roc_core/align_ops.h]: Fixed.

Comment (nolan-veed) [reply src/internal_modules/roc_core/align_ops.h]: It wasn't. But, in hindsight, I shouldn't have updated that.

**Pull request merged**